production/helm/loki-stack/requirements.yaml

```diff
@@ -2,20 +2,20 @@ dependencies:
 - name: "loki"
   condition: loki.enabled
   repository: "file://../loki"
-  version: "^0.6.0"
+  version: "^0.26.0"
 - name: "promtail"
   condition: promtail.enabled
   repository: "file://../promtail"
-  version: "^0.6.0"
+  version: "^0.19.2"
 - name: "fluent-bit"
   condition: fluent-bit.enabled
   repository: "file://../fluent-bit"
-  version: "^0.0.1"
+  version: "^0.0.5"
 - name: "grafana"
   condition: grafana.enabled
-  version: "~3.8.15"
+  version: "~4.6.2"
   repository:  "https://kubernetes-charts.storage.googleapis.com/"
 - name: "prometheus"
   condition: prometheus.enabled
-  version: "~9.3.0"
+  version: "~10.4.0"
```

Comment:
Last change, I'd like to leave Prometheus as is. Not sure why but the helm upgrade fails.

Reply:
Actually I did it to avoid back and forth again !